requirements.txt

```diff
@@ -21,7 +21,8 @@ jsonpath-ng==1.5.2
 keyring==17.1.1
 keyrings.alt==3.1.1
 langid==1.1.6
-lxml==4.5.0
+lxml==4.6.3
+multiset==2.1.1
```

Comment:
python has a built-in multiset called `Counter` in the `collections`; you should use that instead; we want to minimize the external dependencies whenever possible